props/saber_fett263_buttons.h

```diff
@@ -251,10 +256,15 @@ SaberFett263Buttons() : PropBase() {}
 
       // EVENT_PUSH
       if (fabs(mss.x) < 3.0 &&
-          mss.y * mss.y + mss.z * mss.z > 120 &&
+          mss.y * mss.y + mss.z * mss.z > 100 &&
```

Comment:
these lines should not be here.
Please sync/pull/merge to resolve.

Reply:
I don't understand, I changed it to 100 on last update, I found it was easier to replicate consistently

Reply:
And did you do a git pull with those changes before creating this PR?

Reply:
yes, this was in the previous PR, this PR was just updates to the comments at the top of the file

Reply:
No. Your master branch is four commits behind mine.

Reply:
ah, sorry, ok I merged from master, it says no conflicts and and no commits behind